alembic/versions/6fdec36ef433_.py

```diff
@@ -0,0 +1,48 @@
+"""empty message
+
+Revision ID: 6fdec36ef433
+Revises: 
+Create Date: 2024-10-31 13:43:25.528984
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '6fdec36ef433'
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('city',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=255), nullable=False),
+    sa.Column('additional_info', sa.String(length=500), nullable=False),
```

Comment:
The `additional_info` column is set to `nullable=False`. Ensure that this column is required for every entry in the `city` table. If not, consider setting `nullable=True` to allow entries without additional information.